src/preset_cli/cli/superset/sync/dbt/command.py

```diff
@@ -160,17 +157,18 @@ def dbt_core(  # pylint: disable=too-many-arguments, too-many-branches, too-many
             "The managed externally feature was only introduced in Superset v1.5."
             "Make sure you are running a compatible version."
         )
-        log_warning(warn_message, UserWarning)
+        _logger.debug(warn_message)
```

Comment:
Changed it to `debug` as it might make more sense (given it might not be a concern).